eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,12 @@
 ﻿<UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- Ignoring the following prebuilts. We do not have SBRP packages for
+         7.0 targeting packs and System.Text.Encoding.CodePages.6.0.0 package.
+         https://github.com/dotnet/source-build/issues/3341 -->
+    <UsagePattern IdentityGlob="Microsoft.AspNetCore.App.Ref/7.0.2" />
+    <UsagePattern IdentityGlob="Microsoft.NETCore.App.Ref/7.0.2" />
+    <UsagePattern IdentityGlob="System.Text.Encoding.CodePages/6.0.0" />
```

Comment:
Any reason this can't be added to SBRP right away before this is PR is merged in?

Reply:
> Any reason this can't be added to SBRP right away before this is PR is merged in?

Good idea - I'll add `System.Text.Encoding.CodePages/6.0.0` sbrp package and update the comment to reference the other issue for 7.0 targeting packs.

Reply:
Will update this file after SBRP PR is merged - https://github.com/dotnet/source-build-reference-packages/pull/591